README.md

```diff
@@ -31,4 +134,44 @@ cd sam2/
 SAM2_BUILD_ALLOW_ERRORS=0 python setup.py develop
 cd checkpoints/
 ./download_ckpts.sh
-```
\ No newline at end of file
+```
+SAM2 only needs to be installed once. 
+
+## Running SAM2
+
+
+When object positional and entry/exit annotations have been collected and saved in a filename_annotations.npy file, and the video frames have been extracted and stored in a frames folder, the main SAM2 workflow can be followed to process the folder of frames and predict masks for every individual annotated. 
+The main SAM2 workflow must be run on a machine with GPU access.
+
+A folder should be set up containing the `annotations.npy` file, the `frames` subfolder, and the necessary scripts for the SAM2 workflow: `main.py`, `sam2_fish_segmenter.py`, `template_configs.yaml`, `utils.py`, and `plot_utils.py`. 
```

Comment:
```suggestion
A folder should be set up containing the `annotations.npy` file, the `frames` subfolder, and the necessary scripts for the SAM2 workflow: `main.py`, `sam2_fish_segmenter.py`, `template_configs.yaml`, `utils.py`, and `plot_utils.py`. All these files can be obtained from the repo's `SAM2_Tracking` directory. In the future, we will make this a Python package, so that transferring files is not necessary. 
```